native/native.tests/testData/codegen/cinterop/kt73565.kt

```diff
@@ -0,0 +1,53 @@
+// TARGET_BACKEND: NATIVE
+@file:OptIn(kotlinx.cinterop.ExperimentalForeignApi::class)
+
+import kotlin.test.*
+import kotlinx.cinterop.*
+
+fun box(): String {
```

Comment:
Please add tests for top-level functions as well.

Reply:
Good point, don't know why i didn't think of that, will add.